invenio_users_resources/records/dumpers/email.py

```diff
@@ -28,6 +28,7 @@ def dump(self, record, data):
         email_visible = record.preferences["email_visibility"]
         if email_visible == "public":
             data[self.field] = email
+            data[self.hidden_field] = email
```

Comment:
We now always index the email address in the hidden field as well so we can make it searchable for admins.